core/domain/feedback_services_test.py

```diff
@@ -325,15 +326,34 @@ def _close_thread(exp_id, thread_id):
 class EmailsTaskqueueTests(test_utils.GenericTestBase):
     """Tests for tasks in emails taskqueue."""
 
-    def test_create_new_task(self):
+    def test_create_new_batch_task(self):
         user_id = 'user'
-        feedback_services.enqueue_feedback_message_email_task(user_id)
+        feedback_services.enqueue_feedback_message_batch_email_task(user_id)
         self.assertEqual(self.count_jobs_in_taskqueue(), 1)
 
         tasks = self.get_pending_tasks()
         self.assertEqual(
             tasks[0].url, feconf.FEEDBACK_MESSAGE_EMAIL_HANDLER_URL)
 
+    def test_create_new_instant_task(self):
+        user_id = 'user'
+        reference = {
+            'exploration_id': 'eid',
+            'thread_id': 'tid',
+            'message_id': 'mid'
+        }
+        reference_domain = feedback_domain.FeedbackMessageReference(
+            reference['exploration_id'], reference['thread_id'],
+            reference['message_id'])
+
+        feedback_services.enqueue_feedback_message_instant_email_task(
+            user_id, reference_domain)
+        self.assertEqual(self.count_jobs_in_taskqueue(), 1)
+
+        tasks = self.get_pending_tasks()
+        self.assertEqual(
+            tasks[0].url, feconf.INSTANT_FEEDBACK_EMAIL_HANDLER_URL)
```

Comment:
Hm. I think it depends on what the point of this test is. My reading is that its point was to ensure that a newly-enqueued task gets transferred correctly, and I think "transferred correctly" means that the data sent along with the task is preserved. So, might be worth checking explicitly.

(The fact that it's checked implicitly elsewhere doesn't really count; the point of tests is to check things explicitly. Someone may change those other tests later and not realize they're meant to check this condition too.)

On the other hand, perhaps your intention was to test something different, in which case the name of the test should perhaps be clarified...